tesseract_environment/include/tesseract_environment/core/environment.h

```diff
@@ -284,8 +284,8 @@ class Environment
   /**
    * @brief Apply command to the environment
    * @param command Command to be applied to the environment
-   * @return true if successful. If returned false, then only a partial set of commands have been applied. Call
-   * getCommandHistory to check. Some commands are not checked for success
+   * @return true if successful. If returned false, then the command have not been applied.
+   * Some type of Command are not checked for success
```

Comment:
I will need to look into this. The environment is second on my list to get to over 90% code coverage and currently working on tesseract_kinematics.